### DISCUSSION TITLE
Customizable render colors coding

### DISCUSSION THREAD

**Discussion by nmathey** [MadeInPierre/finalynx#89] [Ideas]
Hello,

Le fond de mon terminal est initialement noir par défaut hors certains caract!re s'affichent de la même couleur au rendu :)
J'ai du le mettre en gris pour avoir un compromîtes acceptable mais pas top.

<img width="491" alt="Capture d’écran 2023-05-16 à 06 43 05" src="https://github.com/MadeInPierre/finalynx/assets/20896232/b55f9cd4-555d-4fe3-ad18-a80d8916f79f">

Ce serait intéressant de pouvoir paramétrer les différentes couleurs du rendu

**Comment by MadeInPierre** [comment]
C'est prévu dans #77, j'essaye de m'y mettre dès que j'ai du temps, Je t'invite à discuter dans l'issue pour centraliser ^^
